src/iomei.cpp

```diff
@@ -227,7 +227,6 @@ bool MEIOutput::Export()
             assert(page);
 
             m_currentNode = meiDoc.append_child("score");
-            m_currentNode = m_currentNode.append_child("section");
```

Comment:
I think this is just a good idea when outputting the first page.

Now the output of the second page starts with
```xml
<?xml version="1.0"?>
<score>
   <scoreDef xml:id="soop83z" dynam.dist="4.000000vu" xmlns="http://www.music-encoding.org/ns/mei">
      <staffGrp xml:id="skx6r2p">
         <staffGrp xml:id="s9hud18" bar.thru="true">
            <staffDef xml:id="P1-V8s3TJGdWUOHMPpUWs4d5w" n="1" lines="5">
               <label xml:id="lnes803">Violin II</label>
               <clef xml:id="clef-1" shape="G" line="2" />
               <keySig xml:id="key-1" sig="1s" />
               <meterSig xml:id="time-1" count="4" sym="common" unit="4" />
            </staffDef>
         </staffGrp>
      </staffGrp>
   </scoreDef>
   <measure xml:id="measure-0036-10740-9920-1-mdiv-2" n="76">
      <staff xml:id="sdx6g15" n="1">
         <layer xml:id="lgdtrbx" n="1">
            <note xml:id="note-0036-10780-9960-2" dur="2" oct="6" pname="e" stem.dir="down" />
            <note xml:id="note-0036-11020-9970-2" dur="4" oct="6" pname="d" stem.dir="down" />
            <note xml:id="note-0036-11140-9980-2" dur="4" oct="6" pname="c" stem.dir="down" />
         </layer>
      </staff>
   </measure>
```

So there is no section at all and measures are children of score. This is not MEI conformal.

Reply:
Yes, this is a problem. More fundamentally, we are trying to fix something that was never properly designed. One additional issue when exporting not the first page is that the scoreDef is the one at the beginning of the piece, and not the one currently "in force" on the exported page. This can typically have a wrong clef if there is a clef changed in-between. So we can choose between
1. revert these changes and leave the output un-loadable back as before
2. limit the export of a single page to be possible only the first page
3. have somebody having a proper look at what exporting a single page should mean in terms of score / section structure

For now I would go for 2. but I understand this would not be good for you, would it?

Reply:
Yes, we currently use the single page score based output for ALL pages in our app. So 2 would break existing functionality. I would vote for 1, but we could still remove the additional section when exporting the first page.

I would suggest to create an issue for 3 and try to fix this in a later pull request. The fundamental problem is that we only consider the current page. To fix these issues one needs to run through the whole document to be aware of changes in mdiv, section and scoreDef.

Reply:
OK, I reverted the changes. I should behave as before - obviously the issue with the scoreDef was already there and still is.

Regarding the current scoreDef, we can probably rely on Page::m_drawingScoreDef, but if the page has never been rendered when saving, it will be empty. But as you said, this should go into a separate issue.

Reply:
Great, thank you!